src/generic_core/remote-connection.ts

```diff
@@ -296,12 +297,20 @@ var generateProxyingSessionId_ = (): string => {
           pc = bridge.preObfuscation('sockstortc', config, this.portControl_);
           break;
         case 3:
-          log.debug('using bridge with basicObfuscation');
```

Comment:
Would it be clearer and less error-prone to use an enum or constant here, instead of these numeric literals?